draft-ietf-dnsop-svcb-https.md

```diff
@@ -416,7 +417,8 @@ Using AliasMode maintains a separation of concerns: the owner of
 foosvc.example.net can add or remove ServiceMode SVCB records without
 requiring a corresponding change to example.com.  Note that if
 foosvc.example.net promises to always publish a SVCB record, this AliasMode
```

Comment:
```suggestion
foosvc.example.net promises to always publish a SVCB record and has no other
records on the name, this AliasMode
```

(since this replacement with a CNAME can't happen if the ownername has other records such as NS)

Reply:
There is no requirement that `foosvc.example.com` not have other records, even NS.  It can certainly be a zone apex.

I think you mean "and `_8080._foo.example.com` has no other records".  Any name where this is not true would strike me as pretty bizarre, so I don't see why this needs to be mentioned, but if you think it's necessary we can add it.

Reply:
I misread this. Ignore my comment.  :)